src/channel.rs

```diff
@@ -314,6 +314,24 @@ impl ChannelBuilder {
         self
     }
 
+    /// Set default gzip compression level.
+    pub fn default_gzip_compression_level(mut self, level: usize) -> ChannelBuilder {
```

Comment:
We should make this a nightly API as it's not supported by upstream.

Reply:
done